drivers/storage/portworx/component/pvccontroller.go

```diff
@@ -309,17 +309,14 @@ func (c *pvcController) createDeployment(
 		"--leader-elect=true",
 		"--controllers=persistentvolume-binder,persistentvolume-expander",
 		"--use-service-account-credentials=true",
+		"--leader-elect-resource-name=portworx-pvc-controller",
+		fmt.Sprintf("--leader-elect-resource-namespace=%s", cluster.Namespace),
 	}
 
-	if c.k8sVersion.LessThan(k8sutil.K8sVer1_24) {
-		if pxutil.IsOpenshift(cluster) {
-			command = append(command, "--leader-elect-resource-lock=endpoints")
-		} else {
-			command = append(command, "--leader-elect-resource-lock=configmaps")
-		}
+	if pxutil.IsOpenshift(cluster) {
```

Comment:
let's not use the check, just do this for all cluster.

		command = append(command, "--leader-elect-resource-name=portworx-pvc-controller")
		command = append(command, fmt.Sprintf("--leader-elect-resource-namespace=%s", cluster.Namespace))